monitoring/mock_uss/scdsc/routes_scdsc.py

```diff
@@ -65,6 +70,21 @@ def make_uss_report():
 
     return flask.jsonify({"message": "Not yet implemented"}), 500
 
+
+if "msgsigning" in SERVICES:
+
+    @webapp.route(
+        "/mock/msgsigning/.well-known/uas-traffic-management/pub.der", methods=["GET"]
+    )
+    def get_public_key():
+        public_key_file_location = Config.PUBLIC_KEY_PATH
```

Comment:
Based on the comments above, how about either

```suggestion
        # TODO: Implement runtime changes to certificates (see monitoring/uss_qualifier/suites/faa/uft/design/README.md)
        public_key_file_location = "/var/test-certs/messagesigning/mock_faa_pub.der"
```

or

```suggestion
        # TODO: Implement runtime changes to certificates (see monitoring/uss_qualifier/suites/faa/uft/design/README.md)
        public_key_file_location = os.path.join(webapp.config.get(config.KEY_CERT_BASE_PATH), "messagesigning/mock_faa_pub.der"
```

(depending on whether the CERT_BASE_PATH is made configurable in this PR)

Reply:
I went with the latter!